src/components/ConfigEditor/index.tsx

```diff
@@ -60,12 +74,22 @@ const ConfigEditor: React.FC<ConfigEditorProps> = (props) => {
   }, [options, updateOptions]);
 
   return (
-    <div data-testid="azure-data-explorer-config-editor">
-      <ConfigHelp />
+    <>
+      <DataSourceDescription
+        dataSourceName="Azure Data Explorer"
+        docsLink="https://grafana.com/grafana/plugins/grafana-azure-data-explorer-datasource/"
+        hasRequiredFields
+      />
+      {/* TODO add dividers */
+      /* <Divider/>*/}
+      <ConfigHelp options={options} />
```

Comment:
I believe the `Divider` will help/fix this but right now when ConfigHelp is collapsed it stacks right on top of Connection Details.

![image](https://github.com/grafana/azure-data-explorer-datasource/assets/1048831/144ee725-905d-452b-829c-2bed4dcef40d)

Reply:
Yeah the `Divider` will definitely fix this 😊